packages/react-input/docs/input-demo.stories.js

```diff
@@ -48,6 +48,18 @@ demo.argTypes = {
 			type: 'text',
 		},
 	},
+	suffix: {
+		description: 'This property allows you to add a suffix to the input field.',
```

Comment:
@creador-dev, rephrase the prop description like this: "this property allows you to add some text after the input." This way, users will know "suffix" means "after something," because the other way, you were repeating the property name.